src/modules/modal/modal.css

```diff
@@ -24,11 +26,12 @@
 }
 
 .overlay {
-  width: 100%;
-  height: 100%;
+  width: 100vw;
+  height: 100vh;
```

Comment:
this fixes the autoclosable issue I introduced https://github.com/oacore/design/pull/348